### ISSUE TITLE
Unable to download model when the destination folder is set to working directory

### ISSUE THREAD

**Issue by vasudev-sharma** [axondeepseg/axondeepseg#389]
**Describe the bug**
When we set the destination folder  to the working directory (`axondeepseg`) in the [download_model.py](https://github.com/neuropoly/axondeepseg/blob/master/AxonDeepSeg/models/download_model.py) script, it runs into the following bug.

**Here is the entire error log**
```
(ads_env) (base) ➜  axondeepseg git:(master) ✗ download_models
Trying URL: https://osf.io/2hcfv/?action=download&version=5
Downloading: 100%|#######################################################################################################| 26.8M/26.8M [00:04<00:00, 6.11MB/s]
Unzip...
--> Folder created: /Users/vasudevsharma/VScode/Projects/axondeepseg/default_TEM_model
Trying URL: https://osf.io/sv7u2/?action=download&version=5
Downloading: 100%|#######################################################################################################| 36.2M/36.2M [00:03<00:00, 10.9MB/s]
Unzip...
--> Folder created: /Users/vasudevsharma/VScode/Projects/axondeepseg/default_SEM_model
Data downloaded and unzipped succesfully.
SEM model folder already existed - deleting old one.
TEM model folder already existed - deleting old one.
Traceback (most recent call last):
  File "/Users/vasudevsharma/VScode/Projects/axondeepseg/ads_env/lib/python3.7/shutil.py", line 563, in move
    os.rename(src, real_dst)
FileNotFoundError: [Errno 2] No such file or directory: 'default_SEM_model' -> '/Users/vasudevsharma/VScode/Projects/axondeepseg/default_SEM_model'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/vasudevsharma/VScode/Projects/axondeepseg/ads_env/bin/download_models", line 33, in <module>
    sys.exit(load_entry_point('AxonDeepSeg', 'console_scripts', 'download_models')())
  File "/Users/vasudevsharma/VScode/Projects/axondeepseg/AxonDeepSeg/models/download_model.py", line 39, in main
    download_model(destination = "/Users/vasudevsharma/VScode/Projects/axondeepseg")
  File "/Users/vasudevsharma/VScode/Projects/axondeepseg/AxonDeepSeg/models/download_model.py", line 34, in download_model
    shutil.move("default_SEM_model", str(sem_destination))
  File "/Users/vasudevsharma/VScode/Projects/axondeepseg/ads_env/lib/python3.7/shutil.py", line 577, in move
    copy_function(src, real_dst)
  File "/Users/vasudevsharma/VScode/Projects/axondeepseg/ads_env/lib/python3.7/shutil.py", line 263, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/Users/vasudevsharma/VScode/Projects/axondeepseg/ads_env/lib/python3.7/shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: 'default_SEM_model'

```
**Steps to Reproduce the error**

1. Navigate to  [download_model.py](https://github.com/neuropoly/axondeepseg/blob/master/AxonDeepSeg/models/download_model.py) script.
2. set the destination argument to the working directory, that is,
~~~
def main(argv=None):
    download_model(destination = "/Users/vasudevsharma/VScode/Projects/axondeepseg")
~~~
   Destination folder path could either be absolute or relative.
     3. In the CLI, execute `download_models` command

**Comment by vs74** [comment]
Correct me if I'm wrong?  The `download_data ` function in [ads_utils.py](https://github.com/neuropoly/axondeepseg/blob/master/AxonDeepSeg/ads_utils.py) scripts first creates a temporary folder in root directory (`axondeepseg`) and saves the download model there, and then it moves the temporary folder to the respective destination folder. https://github.com/neuropoly/axondeepseg/blob/4d17eec51437479405e7b511f7b5a3dbfddde304/AxonDeepSeg/ads_utils.py#L181-L222

According to me, the reason for the bug is that  when destination folder is set to root the given code below deletes the temporary created folders of models which were in the root directory.
https://github.com/neuropoly/axondeepseg/blob/4d17eec51437479405e7b511f7b5a3dbfddde304/AxonDeepSeg/models/download_model.py#L27-L35

**Comment by mathieuboudreau** [comment]
Was resolved recently in #836
